File: CS/dxSample/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace dxSample {
    public class Person {
        public Person() {
            
        }
        // Fields...
        private DateTime _Birthdate;
        private string _Name;
        private int _ID;

        public int ID {
            get {
                return _ID;
            }
            set {
                _ID = value;
            }
        }


        public string Name {
            get {
                return _Name;
            }
            set {
                _Name = value;
            }
        }

        public DateTime Birthday {
            get {
                return _Birthdate;
            }
            set {
                _Birthdate = value;
            }
        }
    }
}

File: VB/dxSample/Person.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace dxSample
    Public Class Person
        Public Sub New()

        End Sub
        ' Fields...
        Private _Birthdate As Date
        Private _Name As String
        Private _ID As Integer

        Public Property ID() As Integer
            Get
                Return _ID
            End Get
            Set(ByVal value As Integer)
                _ID = value
            End Set
        End Property


        Public Property Name() As String
            Get
                Return _Name
            End Get
            Set(ByVal value As String)
                _Name = value
            End Set
        End Property

        Public Property Birthday() As Date
            Get
                Return _Birthdate
            End Get
            Set(ByVal value As Date)
                _Birthdate = value
            End Set
        End Property
    End Class
End Namespace
